src/components/Table/GridTable.test.tsx

```diff
@@ -2247,7 +2292,7 @@ describe("GridTable", () => {
     const row1_changed: GridDataRow<Row> = { kind: "data", id: row1.id, data: { name: "one", value: 3 } };
     r.rerender(<GridTable columns={columns} rows={[header, row1_changed, row2]} />);
     // And the original row re-rendered
-    expect(row(r, 1).getAttribute("data-render")).toEqual("2");
+    expect(row(r, 1).getAttribute("data-render")).toEqual("3");
```

Comment:
This is also odd, I don't know why the `data` identity changing triggered this to re-render twice... that said, the main goal of the test is to make sure the _other_ row, on line 2297, does not render/stays at 1.

...so...treating this as quizzical and something I want to fix, but overall not a blocker to the PR imo.